docs/reference/method/MongoDBClient-addSubscriber.txt

```diff
@@ -0,0 +1,124 @@
+================================
+MongoDB\\Client::addSubscriber()
+================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\\Client::addSubscriber()
+
+    Registers a monitoring event subscriber with this Client. The subscriber
+    will be notified of all events for this Client.
+
+   .. code-block:: php
+
+      function addSubscriber(MongoDB\Driver\Monitoring\Subscriber $subscriber): void
+
+   This method has the following parameters:
+
+   .. include:: /includes/apiargs/MongoDBClient-method-addSubscriber-param.rst
+
+   .. note::
+
+         If $subscriber is already registered with this Client, this function
+         is a no-op. If $subscriber is also registered globally, it will still
+         only be notified once of each event for this Client.
+
+Errors/Exceptions
+-----------------
+
+.. include:: /includes/extracts/error-invalidargumentexception.rst
```

Comment:
This exception is duplicated in the extension [doc](https://www.php.net/manual/en/mongodb-driver-manager.addsubscriber.php):
> Throws [MongoDB\Driver\Exception\InvalidArgumentException](https://www.php.net/manual/en/class.mongodb-driver-exception-invalidargumentexception.php) if subscriber is a [MongoDB\Driver\Monitoring\LogSubscriber](https://www.php.net/manual/en/class.mongodb-driver-monitoring-logsubscriber.php), since loggers can only be registered globally.

How should I render it here?